frontend/packages/console-shared/src/hooks/select-list.ts

```diff
@@ -6,12 +6,21 @@ export const useSelectList = <R extends K8sResourceCommon>(
   data: R[],
   visibleRows: Set<string>,
   onRowSelected: (rows: R[]) => void,
+  preSelected: string[] = [],
```

Comment:
I'd prefer not to specify default value, because empty array as default will retrigger useEffect on every rerender (its always a new array instance).